creategameprojects.sh

```diff
@@ -0,0 +1,3 @@
+pushd `dirname $0`

```

Comment:
This, and popd, should be unnecessary. Did this not work without them?

Reply:
I took this from the Source SDK 2013 sh, I can remove it